### PR TITLE
dropdown menu to jump to category

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
image

I somehow need to have an offset, so that scrolling area is not up to blue bar, but ends below dropdown menu

TODO
- [x] fix offset
- [x] obey categories (all categories vs. only categories with unchecked items)

@st3iny @susnux maybe you can share a bit of your wisdom? :blue_heart:

### CODE REVIEW

File: src/views/GroceryList.vue [hunk 1]
```diff
@@ -53,24 +67,24 @@
     </div>
     <div>
```
Comment (Antreesy) [inline src/views/GroceryList.vue:68]: From my POV, this div should have styles to achieve what you want:
```css
margin-top: NUM; // to save space for select
height: NUM; // fixed height, so inner content would scroll
overflow-y: auto; // or scroll, something from these two :upside_down_face: 
``

Comment (tobiasKaminsky) [reply src/views/GroceryList.vue:68]: This is awesome. It works, sort of. :tada: 
With height set to a fixed height, it seems to have to scroll bars, one inner, one outer.

Comment (tobiasKaminsky) [reply src/views/GroceryList.vue:68]: When I set 
```
.app-content {
  overflow: unset !important;
}
```
this work perfect.

Comment (tobiasKaminsky) [reply src/views/GroceryList.vue]: Hm. This works good if height fits.
If it on mobile, with not so big screen, it does not show end.

Additionally, dropdown jumps to correct position on Firefox on PC, but misbehaves on Android.

Comment (tobiasKaminsky) [reply src/views/GroceryList.vue]: And another thing:
dropdown opens keyboard on mobile

**Comment by tobiasKaminsky** [comment]
Another/better approach: hide categories, so that all items are A-Z and can be easily found.

**Pull request closed without merging**